webrender/src/hit_test.rs

```diff
@@ -265,6 +268,17 @@ impl HitTester {
                     None => continue,
                 };
 
+                // Don't hit items with backface-visibility:hidden if they are facing the back.
+                if !item.is_backface_visible {
+                    if facing_backwards.is_none() {
```

Comment:
Thanks for fixing this. Can you use `Option::get_or_insert_with` here? I think this could become a one-liner with that. I'm thinking something like this:

```rust
if *facing_backward.get_or_insert_with(|| transform.is_backface_visible()) {
    continue;
}
```

Reply:
Will do, thanks for the suggestion!